src/components/LoginPage.tsx

```diff
@@ -0,0 +1,40 @@
+import { useFirebase } from 'react-redux-firebase'
+import { useNavigate } from 'react-router-dom'
+import { AppBar, Button, Toolbar, Typography } from '@mui/material'
+
+export const LoginPage = () => {
+  const firebase = useFirebase()
+  const navigate = useNavigate()
+
+  const signInWithGoogle = () => {
+    firebase
+      .login({
+        provider: 'google',
+        type: 'popup',
+      })
+      .then(() => {
+        navigate('/dashboard')
+      })
```

Comment:
`.catch` could be helpful also! You can add it in your `logout` handler as well. We probably should show a kind of error message to the user, to let him/her know that something bad happened.

Reply:
`.catch` with console message added for both handlers